### ISSUE TITLE
Describe better how to get it running on IE11

### ISSUE THREAD

**Issue by David-Mulder** [slimjs/slim.js#40]
I ended up needing a Reflect and Symbol polyfill, and it was an absolute pain getting it up and running (still need to do some debugging), but it would be good to document this better.

Notes: Only polyfill that ended up working for me was the one included in the `babel-polyfill` package. Directly including Reflect from both v5 and v6 of `core-js` didn't work, nor did `harmony-reflect` work for me.

**Comment by eavichay** [comment]
Reflect and Symbols aren't part of slim.js library, and not using these as dependencies. I ended up using a simple string instead of symbol, to support the less useful browsers.

Adding babel-polyfill on a transpiled project is a must, but again, it's not part of slim.js library. I think there should be a documentation note regarding support for IE11. Basically, using the standard polyfills does the trick as long as you work with the native (non-es6) package.

WDYT?

**Comment by David-Mulder** [comment]
Well, as I am making a component to be used on a variety of websites rather than a full website minimal script size is important. Right now I accept that old browsers will need a lot of polyfills, but at least into the future I don't think we want those.

Anyway, in the end I did get it working with just the Reflect polyfill from core js and a Symbol polyfill (without babel-polyfill)... but I am confused what's happening in that case that gets webcomponents js to throw errors that Reflect is missing. As it didn't happen with Polymer I assumed that it was caused by slim.js. Will do better testing sometimes this week or weekend.
